static/js/entry/zendesk_widget.js

```diff
@@ -116,6 +117,27 @@ const zendeskCallbacks = {
         }, 100);
       };
     }
+  },
+  ticketSubmissionFormLoaded: () => {
+    const iframe = document.querySelector("iframe.zEWidget-ticketSubmissionForm");
+    const hiddenFieldNames = ["name", "email", "24690866"];
+    // field 24690866 is the MicroMasters program selector
+    const hiddenFields = _.map(
+      hiddenFieldNames,
+      (name) => {
+        return iframe.contentDocument.querySelector(
+          `input[name="${name}"], select[name="${name}"]`
+        )
+      }
+    );
+    for (const field of hiddenFields) {
+      // hide each field
+      field.closest('label').style.setProperty("display", "none", "important");
```

Comment:
The [`Element.closest()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) method returns the closest ancestor of the current element (or the current element itself) which matches the selectors given in parameter. If there isn't such an ancestor, it returns `null`.

It's the native equivalent of jQuery's [`.parents()`](http://api.jquery.com/parents/) method.